lib/declarations.d.ts

```diff
@@ -42,6 +42,8 @@ declare module Server {
 		solutionPath: string;
 		relativePath: string;
 		localFile?: string;
+		disposition: string;
```

Comment:
well, even if the server does not return it, we'll set it to undefined [here](https://github.com/Icenium/icenium-cli/pull/1214/files#diff-5506b20d356bfab833a3f2bb5fca8bc5R85)
So, let's keep it as is. Ignore my comments above :)